agents/ls-csharp/src/main/resources/org.eclipse.che.ls.csharp.script.sh

```diff
@@ -13,7 +13,7 @@ unset PACKAGES
 unset SUDO
 command -v tar >/dev/null 2>&1 || { PACKAGES=${PACKAGES}" tar"; }
 command -v curl >/dev/null 2>&1 || { PACKAGES=${PACKAGES}" curl"; }
-test "$(id -u)" = 0 || SUDO="sudo -E"
+test "$(id -u)" = 0 || test -f ${HOME}/is_arbitrary_user || SUDO="sudo -E"
```

Comment:
For security reasons, by default, a container in OpenShift is run by an arbitrary user. An arbitrary user is a user with a random UID (and not a sudoer in the container).

We use this trick to be able to run a `*scripts.sh` even if the user is an arbitrary user.

File `${HOME}/is_arbitrary_user` is added using the following mechanism: https://github.com/redhat-developer/che-dockerfiles/blob/master/recipes/centos_jdk8/entrypoint.sh#L30